arangod/Cluster/FollowerInfo.cpp

```diff
@@ -303,6 +303,10 @@ bool FollowerInfo::updateFailoverCandidates() {
 #ifdef ARANGODB_ENABLE_MAINTAINER_MODE
     TRI_ASSERT(_failoverCandidates->size() == _followers->size());
     std::vector<std::string> diff;
+    auto followers = std::make_shared<std::vector<ServerID>>(*_followers);
+    auto failoverCandidates = std::make_shared<std::vector<ServerID>>(*_failoverCandidates);
```

Comment:
The original values are stored in shared_ptrs of std::vectors.
Here we are copying them into our own temporary variables, because we want to modify the copies. Fine.
However, why do we need to copy them into shared_ptrs of vectors, and not just vectors?

Reply:
jan, this is MAINTAINER_MODE it is easily readable.

Reply:
I only repaired the broken code of a colleague.

Reply:
I see, but I don't see any contradiction between this being maintainer mode/code of a colleague and making the code more efficient.

Reply:
@kvahed Isn't that a simple and easy, riskfree improvement?

Reply:
this is a riskfree correction. so far the code is wrong! i am fixing it. this is totally fine code. the copy can be circumvented, but makes the code less readable and it is not even close to performance critical. michael just had overseen that the following difference on the containers is undefined if the are not sorted. really, this is fine as is.